pkg/local_workflows/code_workflow.go

```diff
@@ -0,0 +1,46 @@
+package localworkflows
+
+import (
+	"os"
+
+	"github.com/snyk/go-application-framework/pkg/configuration"
+	"github.com/snyk/go-application-framework/pkg/workflow"
+	"github.com/spf13/pflag"
+)
+
+const (
+	codeWorkflowName = "code.test"
+)
+
+// WORKFLOWID_CODE defines a new workflow identifier
+var WORKFLOWID_CODE workflow.Identifier = workflow.NewWorkflowIdentifier(codeWorkflowName)
+
+// InitCodeWorkflow initializes the code workflow before registering it with the engine.
+func InitCodeWorkflow(engine workflow.Engine) error {
+	// register workflow with engine
+	flags := pflag.FlagSet{}
```

Comment:
Issue: I think we will need to list the flags properly since when this is integrated in the CLI, cobra will not accept any argument that is not listed.